### DISCUSSION TITLE
Chaining of states

### DISCUSSION THREAD

**Discussion by SanjidChowdhury** [tailwindlabs/tailwindcss#11136] [Ideas]
Wouldn't it be great to chain multiple classes to a single state? Such as:

Instead of 
`[&>h2]:text-base [&>h2]:font-semibold`

We can do
`[&>h2]:text-base:font-semibold`

**Comment by wongjn** [comment]
There have been many suggestions about adding such a feature – [here's one such response from a maintainer](https://github.com/tailwindlabs/tailwindcss/discussions/9053#discussioncomment-3350470):

> We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314

[Also another thread here](https://github.com/tailwindlabs/tailwindcss/discussions/8337) with more responses from maintainers.
